src/main/java/com/google/graphgeckos/dashboard/api/components/Log.java

```diff
@@ -0,0 +1,31 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+/**
+ * Represents a log of a buildbot.
+ */
+public class Log {
+  // log type
+  private final String type;
+  // full log link
+  private final String link;
+
+  public Log(String type, String link) {
+    this.type = type;
+    this.link = link;
+  }
+
+  /**
+   * @return a type of the log
```

Comment:
A clarifying example might help make it more clear what this type is.
So far I've only seen 'stdin' log entries in examples; giving that as an example would be useful.

A suggestion to improve this (in a later PR) would be to look at other types of logs are possible and documenting them here.